.github/CODEOWNERS

```diff
@@ -76,6 +77,8 @@
 
 # lib
 /lib/AST/                                           @hborla @slavapestov @xedin
+/lib/AST/*Availability*                             @tshortli
```

Comment:
Did you mean to assign Allan to `lib/Sema/*Availability*` as well?

Reply:
Ah yes, thank you!